packages/sirius-web/frontend/sirius-web-application/src/modals/upload-document/UploadDocumentModal.tsx

```diff
@@ -44,33 +46,41 @@ const uploadDocumentMutationFile = gql`
       ... on ErrorPayload {
         message
       }
+      ... on UploadDocumentSuccessPayload {
+        report
+      }
```

Comment:
The request performed by the frontend to the backend is modified, as a result the new architecture needs to be updated too. Starting with its GraphQL schema, application services, etc...